rust/main/hyperlane-base/src/db/storage_types.rs

```diff
@@ -8,15 +8,18 @@ use hyperlane_core::{
 /// Subset of `InterchainGasPayment` excluding the message id which is stored in
 /// the key.
 #[derive(Debug, Copy, Clone)]
-pub(super) struct InterchainGasPaymentData {
+pub struct InterchainGasPaymentData {
+    /// The amount of tokens paid for the gas.
     pub payment: U256,
+    /// The amount of gas paid for.
     pub gas_amount: U256,
 }
 
 /// Subset of `InterchainGasExpenditure` excluding the message id which is
 /// stored in the key.
+#[allow(missing_docs)]
```

Comment:
just wanna make sure this is intentional as it seems like some docs were added

Reply:
I wasn't fully sure what the fields mean here tbh and I preferred under-documenting that mis-documenting